test/test_prototype_transforms_consistency.py

```diff
@@ -639,7 +639,7 @@ def test_random_apply(self, p):
         prototype_transform = prototype_transforms.RandomApply(
             [
                 prototype_transforms.Resize(256),
-                legacy_transforms.CenterCrop(224),
```

Comment:
This was just plain wrong. It never triggered because the input is a single image. Found this while playing with the container flattening.